src/orion/benchmark/task/profet/profet_task.py

```diff
@@ -0,0 +1,556 @@
+import functools
```

Comment:
Functions are classes should be documented.

Reply:
Most of them are now. Marking this as resolved for now.